### ISSUE TITLE
Allow [paths] normalization to run against a single report

### ISSUE THREAD

**Issue by eric-wieser** [coveragepy/coveragepy#1212]
When running coverage against an installed package, I have a config file like:
```ini
[paths]
source =
    src
    */site-packages

[run]
branch = true
source =
    tests
    my_package_name
```
and a working directory like:
```
src/my_package_name/
   foo.py
tests/
   test_foo.py
setup.py
.coveragerc
```

When I run `coverage run -m pytest`, I get coverage information from `coverage report` for filenames like
```
C:\Users\me\AppData\Local\Programs\Python\Python38\Lib\site-packages\my_package_name\foo.py
tests\test_foo.py
```
However, if I run
```bash
mv .coverage .coverage.hack  # I can't combine it without renaming
coverage combine
coverage report
```
Then I the output has the filenames I (and tools like codecov which match filenames against files in a git repo) want:
```
src\my_package_name\foo.py
tests\test_foo.py
```

**Describe the solution you'd like**
Either the `PathAlias` logic should be applied by default when producing a report, or there should be a command-line-flag / configuration option to make it behave this way

**Additional context**
Somewhat related to https://github.com/pytest-dev/pytest-cov/issues/98

**Comment by nedbat** [comment]
Issue #713 turned out to be another example of this.

I'm trying to think of a situation where it would be bad to re-map the paths during reporting... I'd love to avoid having a command-line option for this.

**Comment by nedbat** [comment]
This is now implemented in commit 0818611c.

**Comment by nedbat** [comment]
This is now released as part of [coverage 7.0.0b1](https://pypi.org/project/coverage/7.0.0b1).
